components/Pagination/PaginationLink/index.js

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+
+class PaginationLink extends React.Component {
+  render() {
+    var label = this.props.label;
+    var ariaLabel = this.props.ariaLabel;
+    var href = this.props.href;
+    var page = this.props.page;
+    var className = this.props.className;
+
+    return (
+      <li className={className}>
+        <a href={href} aria-label={ariaLabel} onClick={this.click.bind(this, page)}>
+          {label}
```

Comment:
we could use the fa classes on the link directly. It's not the most orthodox way but it works.